packages/browser/src/index.test.ts

```diff
@@ -93,12 +120,34 @@ describe('LogtoClient', () => {
     });
   });
 
-  describe('signIn', () => {
-    test('window.location should be correct signInUri', async () => {
+  describe('signIn, handleSignInCallback and isAuthenticated', () => {
+    beforeEach(() => {
+      sessionStorage.clear();
+    });
+
+    test('window.location should be correct signInUri just after calling signIn', async () => {
       const logtoClient = new LogtoClient({ endpoint, clientId }, requester);
       await logtoClient.signIn(redirectUri);
       expect(window.location.toString()).toEqual(mockedSignInUri);
```

Comment:
Should test the logtoConfig settigns have properly set in the sign-in URL instead of mock everything.

Reply:
Updated.